pkg/utils/clustername.go

```diff
@@ -1,6 +1,7 @@
 package utils
 
 import (
+	"google.golang.org/genproto/googleapis/api/annotations"
```

Comment:
Looks like unnecessary packages were imported.

Reply:
Let me clean up it.